protocol/messenger_messages.go

```diff
@@ -84,12 +84,84 @@ func (m *Messenger) EditMessage(ctx context.Context, request *requests.EditMessa
 	}
```

Comment:
missed this comment... what do you mean?

Reply:
If you check the code above, lines 56 to 84, it's basically the same thing that you do in your function  
`editMessagesFromAlbum`.

I initially suggested to put the code above inside an `else` below `if message.ContentType == protobuf.ChatMessage_IMAGE `, but I have a better idea.

We can merge the code you did in `editMessagesFromAlbum` and move it to this function.
Basically, we'll do something very similar to what I did for the DeleteMessage for images.
You can see the code here: https://github.com/status-im/status-go/blob/develop/protocol/messenger_handler.go#L1587-L1624

First, I get an array of messages, if it's an image message, I get all images part of the album, otherwise I just put the one message in the array.

Then I perform the delete actions in a loop, similarly to what you did in `editMessagesFromAlbum`.

If you want, you can probably reuse `getMessagesToDelete`, maybe just find another name like `getAllMessagesAssociatedWithThisMessage`.

You can probably improve that function too. It doesn't do the check you did here ` if image != nil && image.AlbumId != ""`

And that's it.

The goal is just to reduce the amount of duplicated code, since both your function and the code above does the same thing basically.

Reply:
I think I got it :)